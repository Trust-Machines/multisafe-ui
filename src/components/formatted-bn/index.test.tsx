import {render} from '@testing-library/react';

import FormattedBN from './index';

test('1 Render with 6 decimals', () => {
    const rendered = render(<FormattedBN bn="1137100" decimals={6}/>);
    expect(rendered.container).toMatchSnapshot()
});

test('2 Render with 8 decimals', () => {
    const rendered = render(<FormattedBN bn="1100090000" decimals={8}/>);
    expect(rendered.container).toMatchSnapshot()
});

