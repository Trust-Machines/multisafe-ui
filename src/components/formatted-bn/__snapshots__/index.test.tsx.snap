// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`1 Render with 6 decimals 1`] = `
<div>
  <span>
    1.1371
  </span>
</div>
`;

exports[`2 Render with 8 decimals 1`] = `
<div>
  <span>
    11.0009
  </span>
</div>
`;
